crates/sui-indexer-alt/migrations/2024-10-15-170316_transactions/up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS kv_transactions
```

Comment:
For a KV table, I think it should key on tx digest, and not include balance changes, which seems to belong to an indexed tx table.

Reply:
Okay, you got me, I was targeting some kind of half-way house here, I can make this more true to the real key-value schema.